client/src/styles/index.scss

```diff
@@ -0,0 +1,14 @@
+// change the theme
+$theme-colors: (
```

Comment:
Used to override bootstrap defaults color schemes, this is where we would place our stuff from Coolors.